### PR TITLE
Change mail logo to material design icon

### PR DESCRIPTION
**Pull request by GretaD**
To do
- [x] Change all the mail icons to img folder
- [x] Remove the unused ones
- [x] Attribution

Fixes https://github.com/nextcloud/mail/issues/6498

### CODE REVIEW

**Comment by jancborchardt** [comment]
All changed to the Material Design icon. While working on it, especially generating out the png versions, I was thinking which files of these are actually still necessary to manually create and which ones are generated by our theming. @juliushaertl do you know, and/or is this documented somewhere in the app dev docs?

**Comment by juliushaertl** [comment]
> All changed to the Material Design icon. While working on it, especially generating out the png versions, I was thinking which files of these are actually still necessary to manually create and which ones are generated by our theming. @juliushaertl do you know, and/or is this documented somewhere in the app dev docs?

We still need them for cases where ImageMagick is not available or limited. E.g. it is quite common these days to have disable SVG reading by imagemagick in distributions by default, as this may bring additional risk with malicious files.

**Comment by ChristophWurst** [comment]
Do we even use the icons changed with the latest commit https://github.com/nextcloud/mail/pull/6944/commits/505d682991a4ac55d15bc7750236a76b62a561b6? I think those are unused with your recent changes that dropped all custom icons.

**Review by JuliaKirschenheuter — COMMENTED** [review]
should all *.svg and *.png be commited / merged?

**Review by ChristophWurst — APPROVED** [review]
Changes look good

File: img/LICENSES.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Licenses
+### mail.svg
```
Comment (ChristophWurst) [inline img/LICENSES.md:2]:
```suggestion
# Licenses
## mail.svg
```

same for the rest of the doc

**Comment by GretaD** [comment]
> should all *.svg and *.png be commited / merged?

im not sure if i understand what refer with all, but in other apps we only do the svg, if the png is the same icon.

**Comment by jancborchardt** [comment]
@ChristophWurst no, they are not used anymore and can be removed. @GretaD you wanted to remove them right?

**Review by ChristophWurst — APPROVED** [review]

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Squash

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**